### PR TITLE
[14.0][MIG] fieldservice_google_map

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**

### CODE REVIEW

File: fieldservice_google_map/views/fsm_order.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    </t>
+                </templates>
+            </google_map>
+        </field>
+    </record>
+
+    <record id="fieldservice.action_fsm_dash_order" model="ir.actions.act_window">
+        <field name="name">Orders</field>
+        <field name="res_model">fsm.order</field>
+        <field name="view_mode">kanban,timeline,tree,google_map,calendar,form</field>
```
Comment (brian10048) [inline fieldservice_google_map/views/fsm_order.xml]:
```suggestion
        <field name="view_mode">kanban,tree,google_map,calendar,form</field>
```
timeline view was put into its own module

**Comment by max3903** [comment]
@AmmarOfficewalaSerpentCS Can you please fix this one?

**Comment by Freni-OSI** [comment]
Can you please review this PR?
cc: @max3903 @brian10048

**Review by max3903 — APPROVED** [review]

**Pull request merged**